content/sensu-go/6.5/release-notes.md

```diff
@@ -88,6 +89,39 @@ PATCH versions include backward-compatible bug fixes.
 Read the [upgrade guide][1] for information about upgrading to the latest version of Sensu Go.
 
 ---
+## 6.5.0 release notes
+
+**October 12, 2021** &mdash; The latest release of Sensu Go, version 6.5.0, is now available for download.
+
+Insert Release Synopsis.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.5.0.
+
+**NEW FEATURES:**
+
+- Added core/v2.Pipeline resource for configuring event pipelines.
+- Added sensuctl commands for pipeline list, info, and delete.
```

Comment:
I think you can combine these two items.

```suggestion
- New [pipelines][] resource allows you to specify event filters, mutators, and handlers in a single workflow instead of listing filters and mutators in handler definitions. You can reference pipelines in your check definitions. The [/pipelines API endpoint][] provides HTTP access for retrieving pipeline data and configuring pipelines, and you can use [sensuctl][] to manage pipelines.
```